src/Controllers/MoviesController.cs

```diff
@@ -33,10 +33,39 @@ public IActionResult Create(MovieViewModel mvm)
                 UnitOfWork.Movies.Add(new Movie { Title = mvm.Title, ReleaseDate = mvm.ReleaseDate, Genre = mvm.Genre, Price = mvm.Price });
                 UnitOfWork.Complete();
             }
-            return RedirectToAction("Index", "Home");
+            return RedirectToAction("Index", "Movies");
         }
 
         [HttpGet("Index")]
         public IActionResult Index() => View(UnitOfWork.Movies.GetAll().Select(m => new MovieViewModel { Id = m.Id, ReleaseDate = m.ReleaseDate, Title = m.Title, Genre = m.Genre, Price = m.Price } ).ToList());
+    
+        [HttpGet("Edit")]
+        public IActionResult Edit(int Id)
+        {
+            var movie = UnitOfWork.Movies.Get(Id);
+            if (movie != null)
```

Comment:
here we can use:
`if (movie != null)`
`     return View(new MovieViewModel { . . . } );`
`else`
`    return NotFound()`